runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -244,10 +244,18 @@ private void restoreWidgetDimensions()
 
 			if (widget != null)
 			{
-				widget.setRelativeX(widget.getOriginalX());
-				widget.setRelativeY(widget.getOriginalY());
-				widget.setHeight(widget.getOriginalHeight());
-				widget.setWidth(widget.getOriginalWidth());
+				if (widget == client.getWidget(WidgetInfo.FIXED_VIEWPORT_SUB_TAB_CONTAINER))
```

Comment:
what is this for?

Reply:
There is no originalHeight and orginalWidth so if I don't exclude it, it will change the width and height to 0.

Reply:
Could replace it with:
if (widgetOffset.getHeight() == null && widgetOffset.getHeight() == null)